string action
PlannedTrajectory arm_trajectory
